src/test/java/io/neonbee/endpoint/raw/RawEndpointTest.java

```diff
@@ -173,6 +183,35 @@ void testSettingAdditionalHeaders(VertxTestContext testContext) {
                 })));
     }
 
+    @Test
+    @DisplayName("RawDataEndpointHandler should support Brotli compression")
+    void testBrotliCompression(VertxTestContext testContext) {
+
+        NeonBee neonBee = getNeonBee();
+        WebClientOptions opts = new WebClientOptions().setDefaultHost("localhost")
+                .setDefaultPort(neonBee.getOptions().getServerPort());
+
+        HttpRequest<Buffer> request = WebClient.create(neonBee.getVertx(), opts)
+                .request(HttpMethod.GET, String.format("/raw/%s/%s", NEONBEE_NAMESPACE, "Brotli"))
+                .putHeader("Accept-Encoding", "br");
+
+        deployVerticle(new BrotliResponseVerticle())
+                .compose(s -> request.send())
+                .onComplete(testContext.succeeding(response -> testContext.verify(() -> {
+
+                    assertThat(response.getHeader("Content-Encoding")).isEqualTo("br");
+
+                    Buffer compressed = response.body();
+                    // Decompress using BrotliInputStream
+                    BrotliInputStream brotliStream =
+                            new BrotliInputStream(new ByteArrayInputStream(compressed.getBytes()));
+                    byte[] decompressedBytes = brotliStream.readAllBytes();
+                    String decompressedText = new String(decompressedBytes, StandardCharsets.UTF_8);
```

Comment:
I don't understand how this test ensures that NeonBee encodes a Brotli response, since you do the decoding yourself here and as far as I know this shouldn't be necessary anymore if NeonBee accepts Brotli responses.